src/foundry/foundry.js/pixi/containers/placeableObjects/token.d.ts

```diff
@@ -466,9 +411,9 @@ declare class Token extends PlaceableObject<Token.Data> {
 
   /**
    * Toggle the visibility state of any Tokens in the currently selected set
-   * @returns A Promise which resolves to the updated Scene
+   * @returns A Promise which resolves to the updated Token documents
    */
-  toggleVisibility(): Promise<Scene>;
+  toggleVisibility(): Promise<TokenDocument[]>;
```

Comment:
`TokenDocument` does not exist currently. Where should this class be placed?

Reply:
`TokenDocument` extends `CanvasDocumentMixin`, which extends `ClientDocumentMixin`, so it should go under `src/foundry/foundry.js/clientDocuments/canvasDocuments/tokenDocument.d.ts`

Reply:
I don’t think we have `CanvasDocument` yet, do we? So I think for `TokenDocument` to be typed correctly, I think we first need to address that.